src/autorest.bicep/src/schema-generator.ts

```diff
@@ -0,0 +1,729 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { AutorestExtensionHost, Channel } from '@autorest/extension-base';
```

Comment:
FYI a lot of the quirks in this file are due to trying to maintain bug-for-bug compatibility with the current generator. I initially worked on this by comparing the output and iterating to match it.